pkg/virt-operator/resource/apply/reconcile.go

```diff
@@ -125,7 +132,7 @@ const (
 )
 
 // Merge all Tolerations, Affinity and NodeSelectos from NodePlacement into pod spec
-func InjectPlacementMetadata(componentConfig *v1.ComponentConfig, podSpec *corev1.PodSpec) {
+func InjectPlacementMetadata(componentConfig *v1.ComponentConfig, podSpec *corev1.PodSpec, nodePlacementOption DefaultInfraComponentsNodePlacement) {
```

Comment:
nit: It would be nice expressing the intent to pass the node-plac.-opt. to this function in follow up commit in the commit message

Reply:
sure, I can do that